### ISSUE TITLE
Google translation fails in case value is empty string - should just be skipped and not send for translation

### ISSUE THREAD

**Issue by slavap** [sxtfv/flutter_arb_translator#6]
```json
{
  "findTitle": "",
  "findTopMsg": "",
  "findBottomMsg": ""
}
```
google translation fails if the requested value for translation is empty.

Also in case of:
```
"capacityInfo": "{capacities} {childBeds, plural, zero{} other{Child beds: {childBeds}}}"
```
it sends empty value for zero, though it should be just skipped as well.

**Comment by sxtfv** [comment]
Okay, I will fix

**Comment by sxtfv** [comment]
@slavap I just pushed an update (1.0.10) which fixes this and #5 
Hope this package becomes more useful for you

**Comment by slavap** [comment]
@sxtfv Thank you, works fine now.
